server/user.js

```diff
@@ -0,0 +1,85 @@
+const { TimeLogger } = require("../src/util");
+const { R } = require("redbean-node");
+const { UptimeKumaServer } = require("./uptime-kuma-server");
+const server = UptimeKumaServer.getInstance();
+const io = server.io;
+
+/**
+ * Send list of users to client
+ * @param {Socket} socket Socket.io socket instance
+ * @returns {Promise<Bean[]>}
+ */
+async function sendUserList(socket) {
+    const timeLogger = new TimeLogger();
+
+    let result = [];
+    let list = (await R.findAll("user")).map(({ id, username, active }) => ({
```

Comment:
this seems kind of hacky. Would `getAll` be more elegant?

Reply:
I don't find `getAll` in `redbean-node` documentation 😞

Reply:
See https://redbean-node.whatsticker.online/Query

Reply:
Ok, thanks for the documentation. But what is the benefit of that change? Isn't `findAll` producing the same query at the end?

Sorry for what is probably a stupid (and a bit off topic) question, but what is the correct use case of `findAll` exactly? 🤔

Reply:
Not exactly
- `findAll` => `Select * from user` -> reduce variables with map
- `getAll` => `Select id, username, active from user`

=> `findAll` does return all fields, while `getAll` does not

Reply:
Got it! Excellent suggestion, thanks! Will do it 👌🏼

Reply:
For the same reason, also switching on `R.getRow` on L.44 in `getUser` function.

Reply:
Done, see last commit